Makefile

```diff
@@ -174,7 +174,14 @@ build: fmt vet ## Build manager binary.
 
 .PHONY: run
 run: manifests fmt vet ## Run a controller from your host.
-	go run ./main.go
+	go run \
+		-ldflags " \
+			-X 'main.OperatorVersion=$(BUILD_VERSION)' \
+			-X 'main.McadVersion=$(MCAD_VERSION)'  \
+			-X 'main.InstaScaleVersion=$(INSTASCALE_VERSION)' \
+			-X 'main.BuildDate=$(BUILD_DATE)' \
+		" \
+		./main.go
```

Comment:
So this will only set these values when `make run` is used to run the code locally. I believe the root of this issue is actually the Dockerfile. It directly calls `go build` rather than using the make directive specified in this file. 

If you change the dockerfile to use `make build` https://github.com/project-codeflare/codeflare-operator/pull/495/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R164-R173 I think it will solve the issue.